lib/recipes/import/taste_of_home.rb

```diff
@@ -3,26 +3,34 @@ class Import
     class TasteOfHome < Base
       def recipe_name
         document
-          .css('h1.recipe-title')
+          .css('h2.recipe-title')
           .text
       end
 
       def recipe_yield
         document
-          .css('div.makes p')
-          .text
+          .css('div.yield span')
+          .text[/(\d+ \w+)/, 1]
       end
 
       def recipe_prep_time
-        total_time[/Prep: (\d+ \w+)/, 1]
+        prep_time = total_time.index('Prep Time')
+        return nil unless prep_time
+
+        total_time[prep_time + 1]
       end
 
       def recipe_cook_time
-        total_time[/(?:Cook|Bake): (\d+ \w+)/, 1]
+        cook_time = total_time.index('Cook Time')
+        return nil unless cook_time
+
+        total_time[cook_time + 1]
       end
 
       def recipe_total_time
-        total_time[%r{Prep/Total Time: (\d+ \w+)}, 1]
+        document
+          .css('div.time-unit')
+          .text[/(\d+ \w+)/, 1]
       end
```

Comment:
Looks like this is technically grabbing the cook time. We aren't actually storing this in the database. I think we could probably just remove this method.